internal/guts_cli/guts_cli.go

```diff
@@ -114,19 +114,28 @@ func ReadPageAndHWMSize(path string) (uint64, common.Pgid, error) {
 
 // GetRootPage returns the root-page (according to the most recent transaction).
 func GetRootPage(path string) (root common.Pgid, activeMeta common.Pgid, err error) {
+	m, id, err := GetActiveMetaPage(path)
+	if err != nil {
+		return 0, 0, err
```

Comment:
I don't think this is a problem, but the current func returns the id with the respective error

```suggestion
		return 0, id, err
```

Reply:
Updated & thx